lib/common_test/src/test_server_ctrl.erl

```diff
@@ -2498,12 +2501,12 @@ run_test_cases_loop([{conf,Ref,Props,{Mod,Func}}|_Cases]=Cs0,
 			%% this is an end conf for a top level parallel group,
 			%% collect results from the test case processes
 			%% and calc total time
-			OkSkipFail = handle_test_case_io_and_status(),
+			{OkSkipFail, Time} = handle_test_case_io_and_status(),
 			ok = file:set_cwd(filename:dirname(get(test_server_dir))),
 			After = ?now,
 			Before = get(test_server_parallel_start_time),
 			Elapsed = timer:now_diff(After, Before)/1000000,
-			put(test_server_total_time, Elapsed),
+			put(test_server_test_functions_execution_time, get(test_server_test_functions_execution_time) + Time),
```

Comment:
please setup your OTP IDE, and try to limit number of characters in line to at least 100. some prefer 80.